oauthlib/oauth2/rfc6749/clients/web_application.py

```diff
@@ -87,9 +97,10 @@ def prepare_request_uri(self, uri, redirect_uri=None, scope=None,
         """
         scope = self.scope if scope is None else scope
         return prepare_grant_uri(uri, self.client_id, 'code',
-                                 redirect_uri=redirect_uri, scope=scope, state=state, **kwargs)
+                                 redirect_uri=redirect_uri, scope=scope, state=state, code_challenge=code_challenge,
+                                 code_challenge_method=code_challenge_method, **kwargs)
 
-    def prepare_request_body(self, code=None, redirect_uri=None, body='',
+    def prepare_request_body(self, code=None, redirect_uri=None, code_verifier=None, body='',
```

Comment:
Generally, it is better to add new arguments after all existing kwargs, in case someone use them as args; to preserve retrocompatibility.
Would you mind moving `code_verifier` just before **kwargs ?